portfolio/src/main/webapp/comments.html

```diff
@@ -25,17 +25,18 @@
 			<a class="menu-item" href="bucketlist.html">Bucket List</a>
 			<a class="menu-item" href="gallery.html">Gallery</a>
       <a class ="menu-item" href="comments.html">Comments</a>
+      <a class ="menu-item" href="map.html">Map</a>
 		</div>
 	</div>
 
 	<div class="box">
 		<h2 class="title" style="text-align: left; font-size: 2em;">Comments Form</h2>
-    <form action="/new-comment" method="POST" style="font-size: 1.4em">
+    <form id="comment-form" action="/new-comment" method="POST" style="font-size: 1.4em">
       <p>Name</p>
       <input type="text" style="line-height: 20px" name="name" placeholder="Enter name"><br>
       <p>Comment</p>
       <textarea style="width:60vw; height: 25vh;" name="comment" placeholder="Enter comment"></textarea><br>
-      <input type="submit" value="Submit" class="button" style="margin-top: 1%; padding-top: 1%;">
+      <input type="submit" value="Submit" class="button" style="margin-top: 1%;">
```

Comment:
We should avoid using inline styling, could you create a css class for this instead?